lib/scrub_output_structure.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require "pathname"
+require "json"
+
+# Sets up a directory structure for storing autoscrub output files:
+#
+# data/
+# - member_data/
+# - - x/
+# - - - log/
+# - - - output/
+#
+# ... where x is the member_id.
+# Subdirs (loaded, log, output, ready_to_load) are generated upon init.
+# They can have dated subdirs.
+#
+# - - - loaded/
```

Comment:
The directory structure looks good to me, and I like having this class to manage it.

Reply:
👍